tests/VirtoCommerce.Platform.Tests/appsettings.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
You better read an Azure connection string from secret rather than include the entire appsettings.json into test project.

Reply:
appsettings.json removed